src/components/ChatEntry.js

```diff
@@ -1,22 +1,46 @@
-import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ sender, body, timeStamp, id, isLiked, onLikeMessage }) => {
+  const chatStyle =
+    sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote';
+
+  const onLikeClick = () => {
+    onLikeMessage({
+      sender: sender,
+      body: body,
+      timeStamp: timeStamp,
+      id: id,
+      liked: !isLiked,
```

Comment:
Something I missed earlier, when you click the heart, does this function only need to update the `liked` property? 

I think it might, in which case we don't need lines 11-14. Think about giving this function only access to the data it absolutely needs to update. When we write code we want to protect data by only giving access where needed.